canonical/jaas.ai
# Tidy up jaas.ai

## Done

- Deleted unused dependencies
- Deleted unused python and JS files
- Deleted unused templates
- Deleted unused endpoints

## QA

- Visit [demo](https://jaas-ai-735.demos.haus/)
- Ensure the main page and all links from it work correctly
